.github/workflows/main-ci.yml

```diff
@@ -0,0 +1,23 @@
+name: Build
```

Comment:
Can this be renamed to "build.yml" or so?

Reply:
so it appears that there is something wrong with using `build.yml` instead of `main-ci.yml` that makes the formatter *complain*

![image](https://user-images.githubusercontent.com/10422110/174204535-0460a1b0-dd26-4c7b-9b74-f43e727e4d1c.png)

![image](https://user-images.githubusercontent.com/10422110/174204569-da08041d-0016-4e75-9cb8-3742d5f2656b.png)

Reply:
weird, I had it named `build.yml` before and it worked fine... I guess `main-ci.yml` is fine, just seems a bit vague